src/settings.h

```diff
@@ -36,25 +36,34 @@
 
 
 /* Classes ********************************************************************/
-class CSettings
+class CSettings : public QObject
 {
+    Q_OBJECT
+
 public:
     CSettings() :
         vecWindowPosMain ( ), // empty array
         strLanguage      ( "" ),
-        strFileName      ( "" ) {}
+        strFileName      ( "" )
+    {
+        QObject::connect ( QCoreApplication::instance(), &QCoreApplication::aboutToQuit,
```

Comment:
That's a good idea. That improves the code. I've just added this commit to the Git master, thanks.